templates/editentity/history.php

```diff
@@ -63,4 +72,16 @@
                 <?php endif; ?>
             </div>
         <?php endforeach; ?>
+    <?php  ?>
+    <div id="history_pagination_controls" style="width: 50%">
+        <br style="clear: both">
+        <?php if (isset($this->data['history_prev_offset'])): ?>
+        <a style="display: inline-block; float: left"
+           href="editentity.php?eid=<?= htmlspecialchars($entity->getEid()); ?>&selectedtab=8&history_offset=<?= $this->data['history_prev_offset'] ?>#history">&leftarrow; Later revisions</a>
```

Comment:
Well, actually in theory it should be 'htmlspecialchars(urlencode())' as it's a URL encoded in HTML. In practice this is always an integer so it doesn't really matter. 
But as a part of a [Defense of depth](http://en.wikipedia.org/wiki/Defense_in_depth_%28computing%29) approach I really should've done it anyway, good catch, thanks!